.github/workflows/postgres_dev.yaml

```diff
@@ -0,0 +1,70 @@
+name: PG dev
+on:
+  schedule:
+    # run daily 20:00 on master branch
+    - cron: '0 20 * * *'
+  push:
+#    branches:
+#      - prerelease_test
+  pull_request:
+jobs:
+  regress:
+    name: PG14
+    runs-on: ubuntu-20.04
+    env:
+      PG_SRC_DIR: pgbuild
+      PG_INSTALL_DIR: postgresql
+      MAKE_JOBS: 2
+      PG_COMMIT: 5028981923
+      CLANG: clang-9
+      LLVM_CONFIG: llvm-config-9
+      CC: gcc
+      CXX: g++
```

Comment:
at least some of those are needed as there are multiple clang and llvm versions installed in the github environment, also defaults change in the github environment so its good to be explicit

Reply:
Yes, I am thinking about these two that are using defaults only. The others are obviously needed since they are not default. This is not important: you can keep them or remove them as you see fit.